demisto_sdk/common/constants.py

```diff
@@ -688,16 +687,19 @@ class PB_Status:
 
 EXTERNAL_PR_REGEX = r'^pull/(\d+)$'
 
-SCHEMA_TO_VALIDATOR = {
-    'integration': REPORT_REGEX,
+SCHEMA_TO_REGEX = {
+    'integration': YML_INTEGRATION_REGEXES,
     'playbook': YML_ALL_PLAYBOOKS_REGEX,
     'script': YML_SCRIPT_REGEXES,
     'widget': JSON_ALL_WIDGETS_REGEXES,
     'dashboard': JSON_ALL_DASHBOARDS_REGEXES,
     'canvas-context-connections': JSON_ALL_CONNECTIONS_REGEXES,
     'classifier': JSON_ALL_CLASSIFIER_REGEXES,
     'layout': JSON_ALL_LAYOUT_REGEXES,
-    'incidentfield': JSON_ALL_INCIDENT_FIELD_REGEXES,
+    'incidentfield': JSON_ALL_INCIDENT_FIELD_REGEXES + JSON_ALL_INDICATOR_FIELDS_REGEXES
+}
+
+FILE_TYPES_PATHS_TO_VALIDATE = {
     'reports': JSON_ALL_REPORTS_REGEXES,
     'reputation': [MISC_REPUTATIONS_REGEX],
     'reputations': [MISC_REGEX]
```

Comment:
As I said, I don't think this is necessary anymore since I believe that @yaakovi removed it from the content repo. Confirm with @yaakovi.
```suggestion
```